CHANGELOG.md

```diff
@@ -1,48 +1,49 @@
-## 2017-03-29 - Supported Release 2.1.4
+# Changelog
 
-### Summary
+All notable changes to this project will be documented in this file.
 
-Small release with bug fixes
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-### Bug Fixes
+## [Unreleased]
 
-- Unabled to start pipes server in PowerShell process on some Windows 2008r2 images ([MODULES-6927](https://tickets.puppetlabs.com/browse/MODULES-6927))
-- Minor updates to PowerShell syntax in README examples.
+### Added
 
-## 2017-12-08 - Supported Release 2.1.3
+- Metadata for supporting Windows Server 2016 ([MODULES-4271](https://tickets.puppetlabs.com/browse/MODULES-4271))
 
-### Summary
+### Fixed
 
-Small release with bug fixes.
+- Upgrade message is improved to make .NET Framework requirements clearer when running PowerShell 2.0 ([MODULES-7011](https://tickets.puppetlabs.com/browse/MODULES-7011))
+- Fix timeout handling when the user specified a timeout parameter value of `0` ([MODULES-7018](https://tickets.puppetlabs.com/browse/MODULES-7018))
 
-### Bug Fixes
+## [2.1.4] - 2017-03-29
 
-- Fix timeouts and zombie process creation ([MODULES-4748](https://tickets.puppetlabs.com/browse/MODULES-4748))
-- Update PowerShell executable name for experimental cross-platform / PowerShell 6 support ([MODULES-6081](https://tickets.puppetlabs.com/browse/MODULES-6081))
+### Fixed
 
-## 2017-07-27 - Supported Release 2.1.2
+- Unabled to start pipes server in PowerShell process on some Windows 2008r2 images ([MODULES-6927](https://tickets.puppetlabs.com/browse/MODULES-6927))
+- Minor updates to PowerShell syntax in README examples.
 
-### Summary
+## [2.1.3] - 2017-12-08
 
-Small release with bugs fixes.
+### Fixed
 
-### Bug Fixes
+- Fix timeouts and zombie process creation ([MODULES-4748](https://tickets.puppetlabs.com/browse/MODULES-4748))
+- Update PowerShell executable name for experimental cross-platform / PowerShell 6 support ([MODULES-6081](https://tickets.puppetlabs.com/browse/MODULES-6081))
```

Comment:
The way this is written sounds more like a feature than a bug - change 'update' to 'corrected the' - this should make it clearer that we're now using the proper exe name.